frontend/src/libs/components/protected-route/protected-route.tsx

```diff
@@ -1,7 +1,7 @@
 import { type ReactNode } from 'react';
```

Comment:
remove changes in this file, you can add authenticatedUserDataStatus === idle check to app component